script/openqa-bootstrap

```diff
@@ -92,6 +92,8 @@ if [ -n "$skip_suse_tests" ]; then
         fi
         chown -R $dbuser: /var/lib/openqa/tests/opensuse/products/sle/needles
     fi
+else
+    /usr/share/openqa/script/fetchneedles
```

Comment:
I actually thought **skip_suse_tests** means both openSUSE and SUSE needles since [poo#129883](https://progress.opensuse.org/issues/129883) is about a generic openQA container that is not specific to any test distribution.

Reply:
Commit message in https://github.com/os-autoinst/openQA/pull/5163/commits/a370919c26ba3aed543227c9bf191b780f188223 suggests, it shoul deal with SUSE specifics...

Reply:
Also, `os-autoinst-distri-opensuse-deps` is installed into the container so I would guess the container was meant to test os-autoinst-distri-opensuse and hence fetching the needles makes sense.

Reply:
I think that `-n` is used wrong.
```
       -n STRING
              the length of STRING is nonzero
```

So currently only of `$skip_suse_tests` is set to eg `1`, the suse tests are run.
So the whole logic seems to be inverted.

Reply:
I think, @okurz meant to use `-z` instead of `-n` in his original PR.

Reply:
Yes, that's also true.

Reply:
Same for this part:
```bash
if [ -n "$skip_suse_specifics" ] && ping -c1 download.suse.de. && (! rpm -q ca-certificates-suse) ; then
```
This should be `-z` as well.

Reply:
For the record, feel free to go for the most straight-forward fix so we get tests to pass again. I commented on [poo#129883](https://progress.opensuse.org/issues/129883#note-8) regarding the original motivation.